package_scripts/start_course.sh

```diff
@@ -3,31 +3,21 @@
 set -euo pipefail
 
 source package_scripts/common.sh
-load_env --require-dot-env
```

Comment:
Yea, I think we added this a while back because we were having trouble with `dotenv`. Like I mentioned above, I didn't realize that the `dotenv-cli` package was required and that you had to prefix any commands that needed the `.env` file with it. They don't mention it anywhere on the `dotenv` Github and it's kind of puzzling why the "CLI" component of that is a separate package. Either way, it works much better  so yea it makes sense to get rid of this.